### PR TITLE
Translate output client scripts into russian

### PR DESCRIPTION
**Pull request by s1lver**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | no

### CODE REVIEW

File: docs/guide-ru/output-client-scripts.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+```javascript
+var yiiOptions = {"appName":"My Yii Application","baseUrl":"/basic/web","language":"en"};
+```
+
+Теперь в вашем JavaScript коде вы можете получить к ним доступ, например: `yiiOptions.baseUrl` или `yiiOptions.language`.
+
+### Передача сообщений перевода <span id="translated-messages"></span>
+
+Вы можете столкнуться со случаем, когда ваш JavaScript должен вывести сообщение, реагирующее на какое-то событие. В приложении, которое работает с несколькими языками, эта строка должна быть переведена на текущий язык приложения.
+Один из способов достичь этого-использовать [интернационализацию](tutorial-i18n.md#message-translation), предоставляемую Yii и передающий результат в код JavaScript.
```
Comment (samdark) [inline docs/guide-ru/output-client-scripts.md]:
```suggestion
Один из способов достичь этого — использовать [интернационализацию](tutorial-i18n.md#message-translation), предоставляемую Yii и передавать результат в код JavaScript.
```

File: docs/guide-ru/output-client-scripts.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+$message = \yii\helpers\Json::htmlEncode(
+    \Yii::t('app', 'Button clicked!')
+);
+$this->registerJs(<<<JS
+    $('#myButton').on('click', function() { alert( $message ); });
+JS
+);
+```
+
+Приведенный выше пример кода использует PHP [синтаксис Heredoc](https://secure.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) для лучшей читаемости. Это также обеспечивает лучшую подсветку синтаксиса в большинстве IDE, поэтому это предпочтительный способ написания встроенного JavaScript, особенно полезный для кода, который длиннее одной строки. Переменная `$message` создается PHP и благодаря [[yii\helpers\Json::htmlEncode|Json::htmlEncode]] содержит строку в допустимом синтаксисе JS, которую можно вставить в JavaScript код, чтобы поместить динамическую строку в вызов функции `alert()`.
```
Comment (samdark) [inline docs/guide-ru/output-client-scripts.md]:
```suggestion
Приведенный выше пример кода использует PHP [синтаксис Heredoc](https://secure.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) для лучшей читаемости. Это также обеспечивает лучшую подсветку синтаксиса в большинстве IDE, поэтому это предпочтительный способ написания встроенного JavaScript, особенно полезный для кода, более длинного чем однострочный. Переменная `$message` создается PHP и благодаря [[yii\helpers\Json::htmlEncode|Json::htmlEncode]] содержит строку в допустимом синтаксисе JS, которую можно вставить в JavaScript код, чтобы поместить динамическую строку в вызов функции `alert()`.
```

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by samdark** [comment]
@s1lver please address additional comments I've left: https://github.com/yiisoft/yii2/pull/17272/files

**Comment by samdark** [comment]
@s1lver do you have time to finish it?

**Comment by s1lver** [comment]
> @s1lver do you have time to finish it?
Я вроде бы внес последние замечания по тексту? Или что-то пропустил?

**Comment by samdark** [comment]
Пропустил: https://github.com/yiisoft/yii2/pull/17272/files

**Comment by s1lver** [comment]
Хорошо, доработаю

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**